etl/steps/data/meadow/oecd/2024-08-21/official_development_assistance.py

```diff
@@ -0,0 +1,182 @@
+    #         "usd_amount_partial_tied_defl",
+    #         "usd_amounttied_defl",
+    #         "usd_irtc",
+    #         "usd_expert_commitment",
+    #         "usd_expert_extended",
+    #         "usd_export_credit",
+    #         "currency_code",
+    #         "commitment_national",
+    #         "disbursement_national",
+    #         "grant_equiv",
+    #         "usd_grant_equiv",
+    #         "short_description",
+    #         "project_title",
+    #         "purpose_name",
+    #         "sector_name",
+    #         "channel_name",
+    #         "channel_reported_name",
+    #         "geography",
+    #         "ld_cflag",
+    #         "ld_cflag_name",
+    #         "expected_start_date",
+    #         "completion_date",
+    #         "long_description",
+    #         "sd_gfocus",
+    #         "keywords",
+    #         "gender",
+    #         "environment",
+    #         "dig",
+    #         "trade",
+    #         "rmnch",
+    #         "drr",
+    #         "nutrition",
+    #         "disability",
+    #         "ftc",
+    #         "pba",
+    #         "investment_project",
+    #         "assoc_finance",
+    #         "biodiversity",
+    #         "climate_mitigation",
+    #         "climate_adaptation",
+    #         "desertification",
+    #         "commitment_date",
+    #         "type_repayment",
+    #         "number_repayment",
+    #         "interest1",
+    #         "interest2",
+    #         "repaydate1",
+    #         "repaydate2",
+    #         "usd_interest",
+    #         "usd_outstanding",
+    #         "usd_arrears_principal",
+    #         "usd_arrears_interest",
+    #         "capital_expend",
+    #         "ps_iflag",
+    #         "psi_add_type",
+    #         "psi_add_assess",
+    #         "psi_add_dev_obj",
+    #     ],
+    #     "index": [
+    #         "year",
+    #         "donor_name",
+    #         "agency_name",
+    #         "crs_id",
+    #         "project_number",
+    #         "initial_report",
+    #         "recipient_name",
+    #         "region_name",
+    #         "incomegroup_name",
+    #         "flow_name",
+    #         "bi_multi",
+    #         "category",
+    #         "finance_t",
+    #         "aid_t",
+    #         "currency_code",
+    #         "short_description",
+    #         "project_title",
+    #         "purpose_name",
+    #         "sector_name",
+    #         "channel_name",
+    #         "channel_reported_name",
+    #         "investment_project",
+    #     ],
+    # },
+}
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    tables = []
+    for dataset, config in DATASETS.items():
+        # Retrieve snapshot.
+        snap = paths.load_snapshot(f"official_development_assistance_{dataset}.{config['file_format']}")
+
+        # Load data from snapshot.
+        tb = snap.read_in_archive(f"{config['file_name']}.{config['file_format_in_archive']}")
```

Comment:
if combining both properties, could go just with

```py
snap.read_in_archive(config["file_name"])
```

I find it easier to read.

Reply:
Thanks. I had it more complicated because before I opened some csvs directly without zips, but I could also apply this.